chemprop/cli/main.py

```diff
@@ -1,16 +1,16 @@
-from configargparse import ArgumentParser
 import logging
-import sys
 from pathlib import Path
+import sys
 
```

Comment:
While we are sorting everything, would we also make sense to remove these extra lines between imports? I think we would still keep a line between the chemprop imports and other imports.

Reply:
Almost all style guides define three sections of imports, which should be defined in the following order at the top of every file with a blank line between them:
1. standard library
2. third party
3. local imports

I personally define a 4th section, "sublocal," e.g., CLI subpackage imports which I view as logically separate from core library (e.g., `chemprop.*`) imports.